JNI-test-scripts/linux/build-dll.sh

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
For new files, we add the Apache v2 license at the top of them for the AdoptOpenJDK projects (see https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/maketest.sh#L2 for example of header for shell script), applies to all new files in this PR

Reply:
Fixed